apps/web/src/common/components/ProfileImage/types.ts

```diff
@@ -0,0 +1,8 @@
+import { SxProps } from '@mui/material'
+
+export interface ProfileImageProps {
+  src?: string
+  userId: number
+  firstName: string
+  sx?: SxProps
```

Comment:
I'm not sure that this will allow to use theme from ThemeProvider or not, so please consider use `SxPropsWithTheme` instead